tiatoolbox/utils/transforms.py

```diff
@@ -184,8 +189,11 @@ def locsize2bounds(location, size):
          height.
 
     Returns:
-        tuple: A tuple of bounds in (left, top, right, bottom) /
-        (start_x, start_y, end_x, end_y) format.
+        tuple: A tuple of bounds:
+          - :py:obj:`int` - left / start_x
+          - :py:obj:`int` - top / start_y
+          - :py:obj:`int` - right / end_x
+          - :py:obj:`int` - bottom / end_y
     """
```

Comment:
Please add a new line at the end.

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/5b0e0ecc1fcb9ce989d907483bef9a2b0b2c7aa5